linux-tools/audit/audit.sh

```diff
@@ -63,7 +63,7 @@ function tc_local_setup()
         AUDIT_VALUE=$(sed 's/.*audit=\([0-9]*\).*/\1/' $pro_cmd)
         AUDIT_DEBUG=$(sed 's/.*audit_debug=\([0-9]*\).*/\1/' $pro_cmd)
         if [[ ("$AUDIT_ENABLE" -eq "0") && ("$AUDIT_VALUE" -eq "0") && ("$AUDIT_DEBUG" -eq "0") ]]; then
-        tc_conf "Audit Services are disable at kernal please check the audit status" || exit 
+        tc_conf "Audit Services are disable at kernal please check the audit status" && exit 
```

Comment:
Typo: "Audit Services are disable at kernal" . Correct it to kernel
Also correct the summary .